assets/app/styles/_components.less

```diff
@@ -108,22 +108,24 @@
           line-height: @line-height-base;
           .align-self(@align: left);
           color: @gray-light;
-          @media (min-width: @screen-sm-min) and (max-width: @screen-sm-max), (max-width: 380px) {
+          @media (min-width: @screen-sm-min) and (max-width: @screen-sm-max) {
```

Comment:
Prevented me from putting "Add Route" beside the ports at mobile resolutions.

<img width="318" alt="openshift_web_console" src="https://cloud.githubusercontent.com/assets/1167259/11381229/ddef61b6-92c8-11e5-9986-b0138d05d5ce.png">